pippo-session-jdbc/src/main/java/ro/pippo/session/jdbc/JDBCSessionDataStorage.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2016 Herman Barrantes.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ro.pippo.session.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import javax.sql.DataSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ro.pippo.core.PippoRuntimeException;
+import ro.pippo.session.SerializationSessionDataTranscoder;
+import ro.pippo.session.SessionData;
+import ro.pippo.session.SessionDataStorage;
+import ro.pippo.session.SessionDataTranscoder;
+
+/**
+ *
+ * @author Herman Barrantes
+ * @since 10/06/2016
+ */
+public class JDBCSessionDataStorage implements SessionDataStorage {
+
+    // Logger
+    private static final Logger LOG = LoggerFactory.getLogger(JDBCSessionDataStorage.class);
```

Comment:
I prefer `log` instead of `LOG` (in all Pippo we use this convention for logger).